lib/cubits/login/login_cubit.dart

```diff
@@ -48,6 +50,7 @@ class LoginCubit extends Cubit<LoginState> {
       );
     }
 
+    sl<FirebaseAnalyticsEventLogging>().loginEvent();
```

Comment:
User hasn't logged in at this point yet. Line should be moved down to around line 60